lib/torrent.js

```diff
@@ -555,11 +612,11 @@ export default class Torrent extends EventEmitter {
 
     // Select only specified files (BEP53) http://www.bittorrent.org/beps/bep_0053.html
     if (this.so && !this._startAsDeselected) {
-      this.files.forEach((v, i) => {
+      for (const file of this.files ) {
```

Comment:
you mass replaced, and broke this, here the `i` from forEach was used

Reply:
good catch, I used a macro and didn't notice it wasn't using `file` but the index